src/lib.rs

```diff
@@ -24,6 +24,7 @@ struct InnerStarReference {
     header_view: HeaderView,
 }
 
+unsafe impl Send for InnerStarReference {}
```

Comment:
We may want to leave a comment here that this is currently only known to be safe because cellranger runs the destructor on the same thread that creates it.

Reply:
`init_star_ref` calls `new StarRef` and `destroy_ref` calls `delete StarRef`
https://github.com/10XGenomics/orbit/blob/bc050dd4cfb4b60bab0a54cf0ad79db902783e45/star-sys/STAR/source/orbit.cpp#L116-L130

Chris wrote…
> If there's no thread-scoped data magic going on, it's probably safe